tools/ci_build/github/azure-pipelines/nuget/templates/test_win.yml

```diff
@@ -59,31 +62,36 @@ jobs:
     parameters:
       packageFolder: '$(Build.BinariesDirectory)\nuget-artifact'
 
+  # TODO: Add .Net Core AnyCPU test task
```

Comment:
Added some TODOs to trak having separate tests for both .Net Core and .Net Framework "AnyCPU" tests